backend/src/main/java/com/benine/backend/video/MJPEGFrameHeader.java

```diff
@@ -0,0 +1,87 @@
+package com.benine.backend.video;
+
+import java.nio.charset.StandardCharsets;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created on 01-06-16.
+ */
+public class MJPEGFrameHeader {
+
+  private int contentlength;
+  private String boundary;
+  private String contenttype;
+
+  private static final String MJPEG_BOUNDARY = "--polycast";
```

Comment:
The MJPEG requires a boundary starting with "--". Each stream already has a boundary, but there is no use in reading that boundary. Just replacing it with any "--BOUNDARY" string reduces the amount of parsing needed per frame. Do you have a suggestion on I can make this not hardcoded?